text/0018-rubygems-governance.md

```diff
@@ -0,0 +1,185 @@
+4. If removal vote fails, access MUST be restored immediately
+
+### 5.2 Emergency Owner Addition
+In cases where owner count falls below operational minimums:
+1. Remaining owners MAY expedite addition process with 1 week notice
+2. Emergency additions require **ordinary resolution**
+3. Emergency-added owners MUST be confirmed through normal process within 6 months
+
+## 6. Voting Procedures
+
+### 6.1 Voting Methods
+1. Votes SHALL be conducted through secure, verifiable means
+2. Secret ballot voting SHOULD be used when requested by any owner
+3. Vote tallies and outcomes SHALL be recorded and shared with all owners
+
+### 6.2 Quorum and Validity
+1. Votes without quorum are invalid and MUST be re-held
+2. Owners MUST make good faith efforts to participate in votes
+3. Extended absence during voting MAY be grounds for inactivity review
+
+### 6.3 Record Keeping
+1. All votes, proposals, and outcomes SHALL be documented
+2. Records SHALL be accessible to all current owners
+3. Historical governance decisions SHALL be preserved
+
+## 7. Governance Changes
+
+### 7.1 Amendment Process
+1. Any owner MAY propose amendments to this governance document
+2. Amendments require **special resolution** (two-thirds supermajority)
+3. Amendment proposals MUST include:
+   - Specific text changes
+   - Rationale for changes
+   - Impact assessment
+4. Amendments follow the same notice and discussion period as removals
+
+### 7.2 Adoption
+1. This governance document MUST be adopted by **special resolution** of current owners
+2. Adoption requires 3 weeks notice and discussion period
+3. Once adopted, all ownership changes MUST follow these procedures
+
+## 8. Conflict Resolution
+
+### 8.1 Good Faith Requirement
+All owners SHALL make good faith efforts to resolve disputes through discussion and compromise before formal votes.
+
+### 8.2 Mediation
+When disputes arise, owners MAY request neutral mediation from respected community members.
+
+### 8.3 Appeals
+Removed owners MAY appeal their removal by:
+1. Requesting reconsideration with new evidence
+2. Addressing the concerns that led to removal
+3. Appeals require **ordinary resolution** to succeed
+
+## 9. Transition Period
+
+### 9.1 Current Owner Review
+Upon adoption of this governance:
+1. All current owners SHALL be reviewed under the new standards
+2. Any owner not meeting activity requirements MAY be subject to removal vote
+3. Transition votes SHALL follow standard procedures with 3 weeks notice
+
+### 9.2 Grandfathering
+1. Current owners in good standing SHALL retain ownership unless removed through proper procedures
+2. Past inactivity periods SHALL NOT automatically trigger removal
+3. Focus SHALL be on future engagement and contribution
+
+## 10. Implementation
+
+This governance document SHALL take effect immediately upon adoption by special resolution of current project owners. All subsequent ownership changes MUST follow these procedures without exception.
```

Comment:
Do we have a defined list of who the "current project owners" are considered for the purposes of this document?